TabLayout/tests/TabGroup-specs.js

```diff
@@ -32,15 +34,17 @@ describe('TabGroup specs', () => {
 				]}
 			/>
 		);
+		const firstTab = screen.getByRole('group').children.item(0);
+		const secondTab = screen.getByRole('group').children.item(1);
+		const thirdTab = screen.getByRole('group').children.item(2);
 
-		const expected = 3;
-		const actual = subject.find('Tab').length;
-
-		expect(actual).toEqual(expected);
+		expect(firstTab).toBeInTheDocument();
+		expect(secondTab).toBeInTheDocument();
+		expect(thirdTab).toBeInTheDocument();
```

Comment:
the test description implies you should check that there is not a 4th item. Can you do that?